paperqa/docs.py

```diff
@@ -848,3 +855,120 @@ async def aquery(  # noqa: PLR0912
         session.context = context_str
 
         return session
+
+    async def load_docs_from_qdrant(
```

Comment:
So we want to keep `Docs` as a lightweight and general object. Let's keep DB provider stuff out of `Docs`.

Can you perhaps:
- Make this a classmethod or staticmethod on the Qdrant entity
- Make a free function that in that module